banzai_floyds/arc_lines.py

```diff
@@ -20,12 +20,6 @@
         'line_source': 'Hg',
         'line_notes': ''
     },
```

Comment:
I removed some lines in the wavelength solution that were blended. It's fine to include blended lines when we are doing the simultaneous fit of all the lines, but the initial centroiding per line to get the initial wavelength solution parameters can have catastrophic failures. I would like to add a category of lines in the future that doesn't centroid initially but does include the line in the full solution fit.

Reply:
Not so far. I would like to wait on that as a future improvement as we get some experience with how the data quality looks.